webapp/src/pages/Game/Game.css

```diff
@@ -0,0 +1,16 @@
+.button-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 30vh;
+    /* Ajusta según tu diseño */
```

Comment:
These comments in Spanish could be changed to English

Reply:
It is now translated